gdb/gdbarch.c

```diff
@@ -3569,7 +3574,8 @@ gdbarch_register_reggroup_p (struct gdbarch *gdbarch, int regnum, struct reggrou
   gdb_assert (gdbarch != NULL);
   gdb_assert (gdbarch->register_reggroup_p != NULL);
   if (gdbarch_debug >= 2)
-    fprintf_unfiltered (gdb_stdlog, "gdbarch_register_reggroup_p called\n");
+    fprintf_unfiltered (gdb_stdlog, "gdbarch_register_reggroup_p(%d) called\n",
+                        regnum);
```

Comment:
Changes to gdbarch.c will have to be submitted separately from the riscv port.  You are printing regnum here, but not the register group name, which might also be useful.

Reply:
Yeah. If they don't like these changes we can just axe them. But I think they help in debugging.
I'll print reggroup as a pointer. Not sure how helpful that is, but it doesn't hurt and it's easy.